test/test-debuglet.ts

```diff
@@ -315,6 +320,40 @@ describe('Debuglet', function() {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should elaborate on inspector warning on 32 bit but not on 64 bit',
+        function(done) {
+      const projectId = '11020304f2934-a';
+      const debug = new Debug(
+          {projectId: projectId, credentials: fakeCredentials});
+      const debuglet = new Debuglet(debug, defaultConfig);
+      let logText = '';
+      debuglet.logger_.info = function(s: string) {
+        logText += s;
+      };
+      nocks.projectId('project-via-metadata');
+      const scope = nock(API)
+                      .post(REGISTER_PATH)
+                      .reply(200, {debuggee: {id: DEBUGGEE_ID}});
+
+      debuglet.once('registered', function(id: string) {
+        assert.equal(id, DEBUGGEE_ID);
+        // TODO: Handle the case where debuglet.debuggee is undefined
+        assert.equal((debuglet.debuggee_ as Debuggee).project, projectId);
+        const arch = process.arch;
+        if (semver.satisfies(process.version, '>=8') &&
```

Comment:
nit: This should use `>=8.5` (or something similar) for the precise version of 8 that introduced the change.